schema.json

```diff
@@ -1344,6 +1375,183 @@
         "type": "color",
         "label": "Text hover color",
         "id": "card--alternate-color--hover"
+      },
+      {
+        "type": "heading",
+        "content": "Image sizes"
+      },
+      {
+        "type": "paragraph",
+        "content": "Please specify a maximum image size for desktop display. We'll automatically scale down the image for smaller devices.",
+        "hideExpression": "model.productSize != 'product'"
+      },
+      {
+        "type": "select",
+        "label": "Main product images",
+        "id": "productSize",
+        "force_reload": true,

```

Comment:
`force_reload` is not needed here. Stencil cli scan all templates and automatically injects the `force_reload` for you